src/utils/url.ts

```diff
@@ -0,0 +1,8 @@
+export function isValidUrl(url: string): boolean {
```

Comment:
I am not sure where I should add tests for functions like these. It looks like all of the tests in the project are end-to-end tests and do not import typescript code from `src`. @lukastaegert I'd be happy to open a separate PR adding unit tests that execute with `mocha -r ts-node/register`.

Reply:
No, please don't. Then we would still need a test that actually uses the option, i.e. an E2E test. I mean that is the point, to test the effect of this option. As e2e tests in Rollup are REALLY fast, there is really no point in writing unit tests that need to be adapted with every refactoring.
A very simple way could be to add a chunking-form test because those tests just snapshot the output directory and that would include the source map. Or you add a sourcemap test, those tests pass you the source map and you make assertions on the generated map. If all else fails, I will need to look into writing a test myself.